src/beamlime/handlers/detector_data_handler.py

```diff
@@ -135,9 +136,29 @@ class DetectorCounts(Accumulator[sc.DataArray, sc.DataArray]):
     def __init__(self, config: Config, detector_view: raw.RollingDetectorView):
         self._config = config
         self._det = detector_view
+        self._toa_range = ConfigValueAccessor(
+            config, 'toa_range', default=None, convert=self._convert_toa_range
+        )
+        self._current_toa_range = None
+
+    def _convert_toa_range(self, value: dict[str, Any] | None) -> None:
+        self.clear()
+        if value is None:
+            return None
+        return (
+            sc.scalar(value['low'], unit=value['unit']).to(unit='ns'),
+            sc.scalar(value['high'], unit=value['unit']).to(unit='ns'),
```

Comment:
Will the unit always be 'ns', or should we instead just store floats and use the units of `data.bins.coords['toa']` in `apply_toa_range` below?

Reply:
The bounds come via a message from Kafka. It is essential to send to unit with this (or you will waste hours debugging). Not sure what is the benefit from your suggestion, recreating the `low` and `high` variables in every single call below? It is just making it a bit slower?